client/shaders/second_stage/opengl_fragment.glsl

```diff
@@ -1,6 +1,13 @@
 #define rendered texture0
 #define bloom texture1
 
+#ifdef GL_ES
+// Dithering requires sufficient floating-point precision
+#if GL_FRAGMENT_PRECISION_HIGH != 1
```

Comment:
> The built-in macro GL_FRAGMENT_PRECISION_HIGH [...] is not defined on systems not supporting highp precision in the fragment language.
> https://registry.khronos.org/OpenGL/specs/es/2.0/GLSL_ES_Specification_1.00.pdf

Since I have no device which doesn't support `GL_FRAGMENT_PRECISION_HIGH`, I tested this by replacing `GL_FRAGMENT_PRECISION_HIGH` with `MY_UNDEFINED_MACRO`. And apparently such use of an undefined macro results in an error:

```
ERROR[Main]: Irrlicht: GLSL shader failed to compile
ERROR[Main]: Irrlicht: ERROR: 0:12: '' :     GLSL compile error:  unresolved identifier
ERROR: 0:12: '' :     GLSL compile error:  unexpected tokens following an #if or #elif.
ERROR: 2 compilation errors.  No code generated.
ERROR[Main]: generate_shader(): failed to generate "second_stage", addHighLevelShaderMaterial failed.
```

So I think we have to use this instead:

```suggestion
#ifndef GL_FRAGMENT_PRECISION_HIGH
```

Reply:
I agree. I've overlooked this.